app/controllers/submit.js

```diff
@@ -388,8 +403,10 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
     originalPublicationDateChanged: computed('model.originalPublicationDate', 'basicsOriginalPublicationDate', function () {
         const basicsOriginalPublicationDate = this.get('basicsOriginalPublicationDate');
         const modelOriginalPublicationDate = this.get('model.originalPublicationDate');
-        return (basicsOriginalPublicationDate || modelOriginalPublicationDate)
-            && basicsOriginalPublicationDate !== modelOriginalPublicationDate;
+        return (basicsOriginalPublicationDate && !modelOriginalPublicationDate)
+        || (modelOriginalPublicationDate && !basicsOriginalPublicationDate)
+        || (((basicsOriginalPublicationDate && modelOriginalPublicationDate)
+        && modelOriginalPublicationDate.getTime() !== basicsOriginalPublicationDate.getTime()));
```

Comment:
Logic changed to compare getTime() to getTime().  If you changed the date, then changed it back, the UI would reflect that there were unsaved changes, when there were not.